ui/public/src/install-ses-lockdown.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import 'ses/dist/ses.umd'; // adds lockdown, harden, and Compartment
```

Comment:
Okay. And this can be `import 'ses'` when I release a version with `"module": "./dist/ses.umd.js"` in `package.json`?

Reply:
Yes, I think so.